config/routes.rb

```diff
@@ -0,0 +1,22 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+ devise_for :users
+
+  post 'api/v1/auth/login', to: 'authentication#login'
+  namespace :api do
+    namespace :v1 do
+       resources :users
+       resources :cars do
+       resources :reservations
```

Comment:
- I understand you're not using all endpoints for each controller, I kindly invite you to use the `:only` command, to declare what are the endpoints you will use for each controller. 🐔